components/dashboard/src/onboarding/StepOrgInfo.tsx

```diff
@@ -169,7 +169,7 @@ export const StepOrgInfo: FC<Props> = ({ user, onComplete }) => {
                 <TextInputField
                     value={jobRoleOther}
                     onChange={setJobRoleOther}
-                    placeholder="Please specify"
+                    placeholder="Please share (optional)"
                     hint={
                         jobRole === JOB_ROLE_OTHER
                             ? "Please select the role that best describes the type of work you'll use Gitpod for"
```

Comment:
**Suggestion (non-blocking):** I'd suggest to remove this help text. I don't see what additional context it provides a user.

---

If we do decide to keep, I'd suggest...

**Suggestion:** Remove the "work you'll use Gitpod for", part as: 

- User likely doesn't really know what Gitpod is
- We have the other field later on for the purpose



```suggestion
                            ? "Please select the role which best describes your profession"
```

Reply:
Good point, I'll drop the hint completely from this.